### DISCUSSION TITLE
Merging is blocked - Merging can be performed automatically with 1 approving review.

### DISCUSSION THREAD

**Discussion by ashishjullia** [community/community#67622] [Pull Requests]
### Select Topic Area

Bug

### Body

I've the following set of settings/rules enabled on my repo:

![image](https://github.com/community/community/assets/11252951/a81c3327-c059-43ab-886f-6f687dc1cea6)

![image](https://github.com/community/community/assets/11252951/9ffaf7f8-8e93-4235-8d8f-f98c9381141c)

![image](https://github.com/community/community/assets/11252951/bd1b8b94-117d-41d2-9352-e409c27dfee8)

And on my current PR, I have 1 approving review and other checks passed but it is still not allowing me to merge it, what's the issue here? Am I missing something?
![image](https://github.com/community/community/assets/11252951/a1737f41-da51-43d4-b3ef-8410a145373a)

1 Approved out of 2:
![image](https://github.com/community/community/assets/11252951/4bba6929-95af-405e-b073-0cd31d4dbb19)

**Comment by TWiStErRob** [comment]
Read the description of the top red "Review required". I think there's an extra "reviewable push" that wasn't approved yet.

**Reply by ashishjullia** [reply]
@TWiStErRob Nope, I checked that, there aren't any new changes to the "the review I got is on the most recent commit", I double-checked it, and I've checked the rules again, I don't know what is wrong here.

**Reply by ashishjullia** [reply]
So, I checked it again and changed the level or permission for the reviewers on the repo.

From Read -> Triage == rules are not working (gray check marks)

From Triage -> Write == rules are working (green check marks)
![image](https://github.com/community/community/assets/11252951/f1fe4df2-f5a7-446f-aca8-f0814cca0284)

This solved the issue but the information on the github's blog is misleading: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews

![image](https://github.com/community/community/assets/11252951/e1b54b25-99f6-4ad3-bf0b-d11c8f74b267)


LMK, if I'm missing something @TWiStErRob

**Reply by TWiStErRob** [reply]
That's a nice find, they should somehow point this out in the error message. So that's an enhancement request.

I think that last sentence is not a two way logical equivalence as "whether" suggests. Based on your and my experience it should be "Approvals with a gray checkmark do not count towards the required approvals of branch protection rules."

Note that GitHub docs live in a public repo, you can report an issue or a pull request to get it clarified.

Btw, I'm not affiliated with GitHub, just a user myself.

**Reply by DavidBiesack** [reply]
I find it confusing because Write allow one to "push to this repository" -- it's unclear if that means merge (approved) PRs or if it allows direct commits (`git push`). A member role with clear "Approve PRs" would be better (safer).
